cd.js

```diff
@@ -0,0 +1,217 @@
+//Iteraciones para hacer la regresión lineal
+const iterations = 5000;
+//Arrays donde se almacenan los datos
+let xvalues = [];
+let yvalues = [];
+let J = Array(iterations);
+let xiterations = Array(iterations);
+
+//Valores inciales de m y b y=mx+b
+let m =0;
+let b =0;
+
+//Función principal
+//Gráfica
+//Muestra los resultados finales
+async function graph() {
+    const TESTER = document.getElementById('tester');
+    //await getData();
+    await getxData();
+    await getyData();

```

Comment:
Me parece que aqui comentaste la linea 18 porque en este caso tu aplicacion esta en modalidad donde el cliente puede entrar los datos. 

Considero que seria bueno que no tengas que comentar codigo para pasar de una version a otra, y que el cliente tuviera control sobre este aspecto. Una manera de lograr esto podria ser preguntarle al usuario si quiere entrar sus propios datos o si quiere que se ejecute la optimizacion en un CSV predeterminado. Luego, si escoge entrar sus propios datos, se pueden habilitar las lineas 19 y 20. Si no, habilitas la linea 18.

Todo lo anterior se puede lograr a partir de condicionales (por ejemplo: `if (something) { ... }`) y prompts (por ejemplo: `const useCSVAsData = window. prompt("Do you wish to use a predetermined CSV as your dataset?");`)

Que opinas?

Reply:
También me pareció buena idea y no se me había ocurrido como, ya quedó listo eso!. No se como se actualice el código o se tenga que volver a subir....